docs/developer-docs/setup/cycles/cycles-ledger.md

```diff
@@ -0,0 +1,154 @@
+
+The biggest difference between the cycles ledger and cycles wallet is that the cycles ledger is a single global ledger canister, where as the cycles wallet canister must be deployed locally to be used. Additionally, the cycles wallet has some downsides that the cycles ledger remedies. Some of these are:
+
+- Cycles wallets consume cycles themselves, meaning developers need to pay cycles to store cycles.
+- Cycles wallets are a complex solution to associate cycles with principal IDs 
+- Since the cycles wallet canister must be deployed locally, if a cycles wallet canister ID is lost, the cycles in that cycles wallet canister are also lost. 
+
+The cycles ledger solves these complications by providing functionality such as allowing principal IDs to hold cycles, simplifying cycle management, and reducing the risk of loosing cycles. 
+
+The cycles ledger complies with the ICRC-2 standard. In addition to standard ledger functionality, the cycles ledger canister also interacts with the NNS canisters and user canisters to provide cycles ledger-specific functions, such as:
+
+- Depositing cycles to a given principal ID.
+- Sending cycles to a given canister ID.
+- Creating new canisters using cycles.
+
+:::caution
+It is important to note that the cycles ledger does not provide the ability to make arbitrary calls to other canisters with cycles, while the cycles wallet can. This is because open call contexts may cause complications with the cycles ledger. 
+
+If you need to make arbitrary calls to other canisters with cycles, a cycles wallet should be loaded with cycles from the cycles ledger, then the cycles wallet should be used. 
+:::
+
+## Obtaining ICP
+
+To use the cycles ledger, first you need ICP tokens. There are a few different ways to obtain ICP tokens, such as:
+
+- Purchasing ICP tokens directly through an exchange that lists ICP tokens available for trade. To see all available places to purchase ICP tokens, check out this [page](https://coinmarketcap.com/currencies/internet-computer/markets/). 
+- Receiving tokens as rewards for participating in the governance of the Internet Computer.
+- Receiving a grant of tokens through the Internet Computer Association (ICA) or the DFINITY Foundation.
+- Receiving tokens as remuneration for providing computing capacity as a node provider.
+
+To obtain ICP, you also need to have an account to transfer those tokens into. To get your account ID, first determine which dfx identity you are using with the command:
+
+```
+dfx identity whoami
+```
+
+If you're using the default identity, you should create a new identity with the command:
+
+```
+dfx identity new identityName
+```
+
+Your identity's seed phrase will be returned. Be sure to save this in a secure location. Then, set this identity to be used by default:
```

Comment:
Does `dfx identity use` really set a new default or does it simply switch to this identity (for the current session)?